src/Physics/Coherent/EventGen/COHPionKinematicsGenerator.cxx

```diff
@@ -994,5 +1048,11 @@ void COHKinematicsGenerator::LoadConfig(void)
                       kinematics::COHImportanceSamplingEnvelope,0.,1,0.,1,2);
   // stop ROOT from deleting this object of its own volition
   gROOT->GetListOfFunctions()->Remove(fEnvelope);
+
```

Comment:
Yes. Effectively at the beginning we started developing using the same algorithm. While we were at that we also adapted to something more similar to the rest. 
Eventually the system became so different and complex that we decided to have the COHGammaKinematicGenerator. So the options were moved and this is what we left behind.
I decided to leave it since imagining future improvements.